pkg/virt-controller/watch/vmi_test.go

```diff
@@ -1605,6 +1607,59 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 
 			controller.Execute()
 		})
+		Context("And the virt-launcher pod has Multus annotations and SRIOV interface", func() {
```

Comment:
```suggestion
		Context("The VMI has an SR-IOV interface and the virt-launcher Pod has network-status annotations", func() {
```

Reply:
fixed but kept the "And .." in the beginning of the Context. I do this to help make the test more readable (should be read as: `when blabla, And The VMI has an SR-IOV interface and the virt-launcher Pod has network-status annotations.`